src/Sentry.AspNetCore/SentryAspNetCoreOptions.cs

```diff
@@ -43,6 +43,20 @@ public bool IncludeRequestPayload
         /// </remarks>
         public RequestSize MaxRequestBodySize { get; set; }
 
+        /// <summary>
+        /// Flush on completed request
+        /// </summary>
+        public bool FlushOnCompletedRequest { get; set; }
+
+        /// <summary>
+        /// How long to wait for the flush to finish, in seconds. Defaults to 2 seconds.
```

Comment:
```suggestion
        /// How long to wait for the flush to finish. Defaults to 2 seconds.
```
It's not 'in seconds' because it's a `TimeSpan`. It's a `struct` to represent a slice of time and you can create an instance from seconds, milliseconds, days, etc.